client/extensions/woocommerce/lib/countries/US.js

```diff
@@ -12,206 +21,257 @@ export default () => {
 				{
 					code: 'AL',
 					name: translate( 'Alabama' ),
+					salesTaxBasis: DESTINATION_BASED_SALES_TAX,
 				},
 				{
 					code: 'AK',
 					name: translate( 'Alaska' ),
+					salesTaxBasis: NO_SALES_TAX,
 				},
 				{
 					code: 'AZ',
 					name: translate( 'Arizona' ),
+					salesTaxBasis: ORIGIN_BASED_SALES_TAX,
 				},
 				{
 					code: 'AR',
 					name: translate( 'Arkansas' ),
+					salesTaxBasis: DESTINATION_BASED_SALES_TAX,
 				},
 				{
 					code: 'CA',
 					name: translate( 'California' ),
+					salesTaxBasis: ORIGIN_BASED_SALES_TAX,
 				},
 				{
 					code: 'CO',
 					name: translate( 'Colorado' ),
+					salesTaxBasis: DESTINATION_BASED_SALES_TAX,
 				},
 				{
 					code: 'CT',
 					name: translate( 'Connecticut' ),
+					salesTaxBasis: DESTINATION_BASED_SALES_TAX,
 				},
 				{
 					code: 'DE',
 					name: translate( 'Delaware' ),
+					salesTaxBasis: NO_SALES_TAX,
 				},
 				{
 					code: 'DC',
 					name: translate( 'District Of Columbia' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'District of Columbia' )` *ES Score: 22*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).